include/utilities/CSV_Reader.h

```diff
@@ -34,8 +34,8 @@ inline std::vector<std::vector<std::string> > CSVReader::getData()
     std::ifstream file(fileName);
 
         if(file.fail()){
```

Comment:
In reviewing this, I think this test may actually want `if (!file.good())`, but I need to parse the docs much more closely. This was noticed while trying to read up whether ...